libpod/lock/shm_lock_manager_linux.go

```diff
@@ -60,6 +61,11 @@ func (m *SHMLockManager) RetrieveLock(id uint32) (Locker, error) {
 	lock.lockID = id
 	lock.manager = m
 
+	if id >= m.locks.GetMaxLocks() {
+		return nil, errors.Errorf("given lock ID %d is too large - this manager only supports lock indexes up to %d",
```

Comment:
Shorten message

"Lock id %d is too large - manager support max %d lock indexes"

Also wrap 
errors.Wrapf(syscall.EINVAL, "Lock id %d is too large - manager support max %d lock indexes"

Reply:
@mheon Fix this and we can get this merged.

Reply:
Done